src/bench_ecmult.c

```diff
@@ -185,14 +186,17 @@ int main(int argc, char **argv) {
     free(pubkeys_gej);
 
     for (i = 1; i <= 8; ++i) {
-        run_test(&data, i, 1);
+        run_test(&data, i, 1, iters);
     }
 
-    for (p = 0; p <= 11; ++p) {
-        for (i = 9; i <= 16; ++i) {
-            run_test(&data, i << p, 1);
+    if (iters > 1) {
```

Comment:
Would be good to add comment why that's useful.

Reply:
Done